src/Symfony/Component/VarDumper/Caster/ReflectionCaster.php

```diff
@@ -74,6 +74,59 @@ public static function castClosure(\Closure $c, array $a, Stub $stub, $isNested)
         return $a;
     }
 
+    public static function castGenerator(\Generator $c, array $a, Stub $stub, $isNested)
+    {
+        return self::castReflectionGenerator(new \ReflectionGenerator($c), $a, $stub, $isNested);
```

Comment:
wouldn't this cause issues on PHP 5.6, which has Generator but not ReflectionGenerator ?